Library/Formula/libmustache.rb

```diff
@@ -0,0 +1,21 @@
+class Libmustache < Formula
+  desc "C++ Implementation of the mustache template library"
+  homepage "https://github.com/jbboehr/libmustache"
+  url "https://github.com/jbboehr/libmustache/archive/v0.3.1.tar.gz"
+  sha256 "c09f1bcfb8b272eac7cad4c651e0810b3f4dde893bd1fb4b1e4f64e545cf465c"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "autoreconf", "-fvi"
+    system "./configure", "--disable-debug",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/mustache", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!